sdk/computefleet/azure-resourcemanager-computefleet/src/test/java/com/azure/resourcemanager/computefleet/ComputeFleetManagerTests.java

```diff
@@ -0,0 +1,284 @@
+                                                                                        .withPrimary(true)
+                                                                                        .withSubnet(
+                                                                                            new ApiEntityReference()
+                                                                                                .withId(network.subnets().get("default").id())
+                                                                                        )
+                                                                                        .withPublicIPAddressConfiguration(
+                                                                                            new VirtualMachineScaleSetPublicIPAddressConfiguration()
+                                                                                                .withName(publicIpName)
+                                                                                                .withSku(
+                                                                                                    new PublicIPAddressSku()
+                                                                                                        .withName(PublicIPAddressSkuName.STANDARD)
+                                                                                                        .withTier(PublicIPAddressSkuTier.REGIONAL)
+                                                                                                )
+                                                                                                .withProperties(
+                                                                                                    new VirtualMachineScaleSetPublicIPAddressConfigurationProperties()
+                                                                                                        .withDeleteOption(DeleteOptions.DELETE)
+                                                                                                        .withPublicIPAddressVersion(IPVersion.IPV4)
+                                                                                                        .withIdleTimeoutInMinutes(9)
+                                                                                                        .withPublicIPPrefix(
+                                                                                                            new SubResource()
+                                                                                                                .withId(publicIpPrefix.id())
+                                                                                                        )
+                                                                                                )
+                                                                                        )
```

Comment:
Does create work without the PublicIPAddressConfiguration?

Reply:
The `PublicIPAddressConfiguration` replaced with `LoadBalancerBackendAddressPools`.